com.microsoft.mrtk.core/Editor/Utilities/InspectorUIUtility.cs

```diff
@@ -541,229 +594,408 @@ public static Enum DrawEnumSerializedProperty(Rect position, SerializedProperty
         }
 
         /// <summary>
-        /// adjust list settings as things change
+        /// Draws a foldout enlisting all components (or derived types) of the given type attached to the passed GameObject.
+        /// Adds a button for adding any of the component (or derived types) and a follow button to highlight existing attached components.
         /// </summary>
-        public static List<ListSettings> AdjustListSettings(List<ListSettings> listSettings, int count)
+        static public bool DrawComponentTypeFoldout<T>(GameObject gameObject, bool isExpanded, string typeDescription) where T : MonoBehaviour
```

Comment:
super nit, since i'm pretty sure this is just moved from elsewhere in this file: but `static` and `public` are swapped from our normal convention

Reply:
Fixed